manifests/redictcluster.pp

```diff
@@ -8,6 +8,7 @@
   Optional[Boolean] $prevent_reboot = false,
   Optional[String] $image = 'registry.redict.io/redict',
   Optional[String] $tag = '7-bookworm',
+  Optional[String] $maxmemory_policy = 'noeviction',
 
```

Comment:
Maybe:
```
Optional[String] $maxmem = undef,
String $maxmemory_policy = 'noeviction',
```

Reply:
how about `Optional[Integer] $maxmem = 0`, is no limit and it is the default for 64-bit systems?